app/assets/stylesheets/argo/icons.scss

```diff
@@ -0,0 +1,31 @@
+// Helper classes for icons
+
+// Spinner
+.glyphicon-spin {
+  -webkit-animation: spin 1000ms infinite linear;
+  animation: spin 1000ms infinite linear;
+}
+
+@-webkit-keyframes spin {
```

Comment:
Should be resolved now by https://github.com/sul-dlss/argo/pull/385/files#diff-cb265949e6e497eb8fdfe94dc79b9909R4